### PR TITLE
Rework create chat avoid react errors

### PR DESCRIPTION
**Pull request by nicodh**
@Simon-Laux I used the same hardcoded fallback color #505050 for pseudo group/contact that was used in AvatarBubble. Not sure if we need to adapt that for theming...

### CODE REVIEW

**Comment by Simon-Laux** [comment]
Ah right that would be a usecase of a default color... anyways avatar color is nothing that will be changable with a theme, because the text is always white and the bg color comes from the core so I think we'll be fine for this one.

File: src/renderer/components/dialogs/CreateChat-Styles.js [hunk 1]
```diff
@@ -105,11 +105,11 @@ export const GroupImageWrapper = styled.div`
 `
 
 export const GroupImage = (props) => {
-  const { groupImage, onSetGroupImage, onUnsetGroupImage } = props
+  const { groupImage, onSetGroupImage, onUnsetGroupImage, ...otherProps } = props
   return (
     <GroupImageWrapper>
       { groupImage && <AvatarImage src={groupImage} onClick={onSetGroupImage} {...props} /> }
-      { !groupImage && <AvatarBubble onClick={onSetGroupImage} {...props}>G</AvatarBubble> }
+      { !groupImage && <AvatarBubble onClick={onSetGroupImage} {...otherProps}>G</AvatarBubble> }
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/CreateChat-Styles.js]: not sure if we should use such a wildcard

Comment (nicodh) [reply src/renderer/components/dialogs/CreateChat-Styles.js]: Well before it passed all properties - now all except onSetGroupImage, onUnSetGroupImage, GroupImage where onSetGroupImage, onUnSetGroupImage are not known properties for AvatarBubble...

Comment (Simon-Laux) [reply src/renderer/components/dialogs/CreateChat-Styles.js]: ok

File: src/renderer/components/dialogs/CreateChat-Styles.js [hunk 2]
```diff
@@ -105,11 +105,11 @@ export const GroupImageWrapper = styled.div`
 `
 
 export const GroupImage = (props) => {
-  const { groupImage, onSetGroupImage, onUnsetGroupImage } = props
+  const { groupImage, onSetGroupImage, onUnsetGroupImage, ...otherProps } = props
```
Comment (Jikstra) [inline src/renderer/components/dialogs/CreateChat-Styles.js]: From where are you getting otherProps?

**Pull request merged**